microsoft-edge/visual-studio-code/microsoft-edge-devtools-extension.md

```diff
@@ -1,188 +1,220 @@
 
-## Opening Microsoft Edge Developer Tools with Visual Studio Code  
+<!-- ====================================================================== -->
+## Opening Microsoft Edge DevTools within Visual Studio Code
 
-To open the tools panel, choose the Microsoft Edge Tools icon from the Activity bar.
+To open the tools panel, in the **Activity Bar**, select the **Microsoft Edge Tools** icon.
 
-The Microsoft Edge Developer Tools for Visual Studio Code extension allows you to launch an instance of Edge or generate a `launch.json` file to automate the debugging workflow. 
+The Microsoft Edge Tools extension for Visual Studio Code allows you to launch an instance of Edge or generate a `launch.json` file to automate the debugging workflow.
 
 :::image type="complex" source="./media/edge-devtools-for-vscode-extension-icon.png" alt-text="Microsoft Edge DevTools for Visual Studio Code extension" lightbox="./media/edge-devtools-for-vscode-extension-icon.png":::
-   Microsoft Edge DevTools for Visual Studio Code extension  
+   Microsoft Edge DevTools for Visual Studio Code extension
 :::image-end:::
 
-Selecting **Launch Instance** opens a browser window and the Edge developer tools in Visual Studio Code.   
+Selecting **Launch Instance** opens a browser window and the Edge developer tools in Visual Studio Code.
 
 :::image type="complex" source="./media/edge-devtools-for-vscode-launch-instance.png" alt-text="Select Launch Instance to open the browser in Visual Studio Code" lightbox="./media/edge-devtools-for-vscode-launch-instance.png":::
-   Select Launch Instance to open the browser in Visual Studio Code  
+   Select **Launch Instance** to open the browser in Visual Studio Code
 :::image-end:::
 
-Use the Developer Tools extension in VS Code to inspect an HTML Element in Microsoft Edge. For example, select **Success!** In the browser and notice that the Elements tool opens with the HTML highlighted. 
+Use the Developer Tools extension in VS Code to inspect an HTML Element in Microsoft Edge. For example, select **Success!** In the browser and notice that the Elements tool opens with the HTML highlighted.
 
-:::image type="complex" source="./media/edge-devtools-for-vscode-elements.png" alt-text="Elements tool open with HTML highlighted" lightbox="./media/edge-devtools-for-vscode-elements.png":::
-   Elements tools open with HTML highlighted  
+:::image type="complex" source="./media/edge-devtools-for-vscode-elements.png" alt-text="Elements tool with HTML highlighted" lightbox="./media/edge-devtools-for-vscode-elements.png":::
+   The **Elements** tool with HTML highlighted
 :::image-end:::
 
-## Using Microsoft Edge Developer Tools in Visual Studio Code  
 
+<!-- ====================================================================== -->
+## Modes for using Microsoft Edge Developer Tools in Visual Studio Code
```

Comment:
@codepo8 How do these modes relate to the headings?  Does each of these 3 modes (list items) correspond to a specific heading below?  If so, which heading is for which mode that's listed here?

Reply:
dup (a result of github PR ui)